engine/jobmaster/dm/task_manager.go

```diff
@@ -40,6 +40,7 @@ var (
 type TaskManager struct {
 	*ticker.DefaultTicker
 
+	jobID        string
```

Comment:
Is this field just a compatible field and only be used in `op`?
May be we can use a `fakeJobID` or `dummyJobID`.

Reply:
This is used to extract the job_id label from the monitor.